homeassistant/helpers/template.py

```diff
@@ -2113,6 +2115,42 @@ def as_timedelta(value: str) -> timedelta | None:
     return dt_util.parse_duration(value)
 
 
+def merge_response(
+    value: ServiceResponse, sort_by: str | None = None, single_key: str | None = None
+) -> list[Any]:
+    """Merge service responses into single list.
+
+    Checks that the input is a correct service response:
+    {
+        "entity_id": {str: Any},
+    }
```

Comment:
I think the docstring should explain the special treatment of lists too

Reply:
I added a line for explaining the list treatment.